pkgs/applications/video/shaka-packager/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, stdenv
+, fetchurl
+, runCommand
+, shaka-packager
+}:
+
+let
+  sources = {
+    "x86_64-linux" = {
+      filename = "packager-linux-x64";
+      hash = "sha256-MoMX6PEtvPmloXJwRpnC2lHlT+tozsV4dmbCqweyyI0=";
+    };
+    aarch64-linux = {
+      filename = "packager-linux-arm64";
+      hash = "sha256-6+7SfnwVRsqFwI7/1F7yqVtkJVIoOFUmhoGU3P6gdQ0=";
+    };
+    x86_64-darwin = {
+      filename = "packager-osx-x64";
+      hash = "sha256-fFBtOp/Zb37LP7TWAEB0yp0xM88cMT9QS59EwW4MrAY=";
+    };
+  }."${stdenv.hostPlatform.system}"
+    or (throw "Unsupported system: ${stdenv.hostPlatform.system}");
+in
+stdenv.mkDerivation rec {
+  pname = "shaka-packager";
+  version = "2.6.1";
+
+  src = fetchurl {
+    url = "https://github.com/shaka-project/shaka-packager/releases/download/v${version}/${sources.filename}";
+    inherit (sources) hash;
+  };
+
+  dontUnpack = true;
+  sourceRoot = ".";
+
+  installPhase = ''
+    runHook preInstall
+
+    install -m755 -D $src $out/bin/packager
```

Comment:
`packager` is too "generic".
I believe we should rename it to `shaka-packager`.
However, it can be problematic because I don't know how other distros name it.

(I always cite the hypothetical case of "someone that uses a script that expects the official naming".)

Edit: only aur, and the name is `packager` indeed.

Reply:
I'm not sure about this. I imagine when people install the package, they expect the tool to be called `packager` rather than anything else. Happy to trust your judgement and make the change if required though.

Reply:
I'm not sure if it's the ~best way to check for existing clashes are, but FWIW `nix run github:mic92/nix-index-database /bin/packager` doesn't indicate any.